src/App.tsx

```diff
@@ -1,17 +1,157 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
+import './App.scss';
+import './components/TodoList/TodoList.scss';
 
 import users from './api/users';
+import todos from './api/todos';
+import { Todos, Users, PrepearedTodos } from './react-app-env';
+import { TodoList } from './components/TodoList/TodoList';
 
 const App: React.FC = () => {
+  const prepearingTodos = (todosList : Todos[], usersList : Users[]) => {
+    return todosList.map(todo => ({
+      ...todo,
+      user: usersList.find(user => todo.userId === user.id)?.name,
+    }));
+  };
+
+  const prepearedTodos: PrepearedTodos[] = prepearingTodos(todos, users);
+
+  const [currentTodos, setCurrentTodos] = useState(prepearedTodos);
+
+  const [todoTitle, setTodoTitle] = useState('');
+  const [todoUserId, setTodoUserId] = useState(0);
+  const [todoIsCompleted, setIsCompleted] = useState(false);
+
   return (
     <div className="App">
-      <h1>Add todo form</h1>
+      <form
+        id="main_form"
+        action="GET"
+        onSubmit={(event) => {
+          event.preventDefault();
+        }}
+      >
+        <input
+          type="text"
+          name="title"
+          id="main_form-name"
+          placeholder="title"
+          onChange={(event => {
+            if (event.target.classList.contains('not_valid')) {
+              event.target.classList.remove('not_valid');
+            }
```

Comment:
don't use classList in react, it is better to use classnames npm library
https://www.npmjs.com/package/classnames